napari/components/add_layers_mixin.py

```diff
@@ -67,7 +123,7 @@ def add_image(
         blending=None,
         visible=True,
         path=None,
-    ):
+    ) -> Union[layers.Image, List[layers.Image]]:
```

Comment:
Ha, I was like, "surely not!", and had to keep expanding the signature until I was proven wrong. 😂